review.js

```diff
@@ -0,0 +1,79 @@
+// jQuery track any click event on page
+$(document).on('click', function() {
+  console.log('clicked');
+});
+
+// Vanilla JS track any click event on page
+document.onclick = (e) => {
+    console.log('clicked');
+};
+
+//======================================================================================
+
+// jQuery tracking everytime apply button is clicked, .click handler
+$('*[data-nav-link="Apply"]').click(function(){
+    console.log('clicked' , $(this).text());
+});
+
+// Vanilla JS track everytime apply button is clicked, .click handler
+document.querySelectorAll('[data-nav-link="Apply"]');
+
+document.getElementById('button').onclick = function() {
+    console.log('clicked');
+}​;​
+
+
+
+// jQuery tracking everytime button with data-nav-link is clicked, .click handler
+$('data-nav-link').click(function(){
+    console.log('clicked' , $(this).text());
+});
+
+// jQuery tracking everytime Calculate Your Loan button is clicked, .click handler
+$('*[data-link-name="Calculate Your Loan!"]').click(function(){
+    console.log('clicked' , $(this).text());
+});
+
+//======================================================================================
+
+// jQuery tracking form submit for subscribe form.submit handler
+$('*[data-form-id="promotion-subscription"]').on('submit', function(e) { 
```

Comment:
rather than this, create one that whenever ANY form is submitted, it logs out the data-form-id attribute.
That's the reason we have the data attr on there so we can surface which form is being submitted.

Reply:
there are 2 jquery methods you can use to grab the contents of a data attribute. do one for each.